lib/ansible/modules/system/lvg.py

```diff
@@ -167,8 +168,9 @@ def main():
         # get pv list
         pvs_cmd = module.get_bin_path('pvs', True)
         if dev_list:
-            pvs_filter = ' || '. join(['pv_name = {0}'.format(x) for x in dev_list + module.params['pvs']])
-            pvs_filter = "--select '%s'" % pvs_filter
+            pvs_filter_pv_name = ' || '. join('pv_name = {0}'.format(x) for x in itertools.chain(dev_list, module.params['pvs']))
```

Comment:
plz remove that whitespace after a dot:
```suggestion
            pvs_filter_pv_name = ' || '.join('pv_name = {0}'.format(x) for x in itertools.chain(dev_list, module.params['pvs']))
```

Reply:
and maybe make it multiline for readability:
```suggestion
            pvs_filter_pv_name = ' || '.join(
                'pv_name = {0}'.format(x)
                for x in itertools.chain(dev_list, module.params['pvs'])
            )
```

Reply:
Strange, the space after the dot passed flake8, didn't aware it is valid python...Amend with the multiline version which is way clearer